js/index.js

```diff
@@ -40,3 +40,125 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+//Nav Anchors
```

Comment:
great job with this little comments to yourself. It really helps show me the story of how working on this went. I love it.